packages/metrics_center/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: metrics_center
-version: 1.0.9
+version: 1.0.10
```

Comment:
Can you please add an update to the changelog as well?

Reply:
Updated!